dashboard/app/assets/stylesheets/application.scss

```diff
@@ -3073,6 +3073,9 @@ button:hover>img {
 button:active {
   border: 1px solid $light_gray !important;
 }
+button.button-active-no-border:active {
+  border: none !important;
```

Comment:
i know why this is needed (and the reason is right above it 😄), but can you add a comment just to make it extra clear?